packages/workit-types/src/camundaBpm/subscriptionOptions.ts

```diff
@@ -29,6 +29,14 @@ export interface ISubscriptionOptions<T = unknown> {
    * A value which allows to filter tasks based on process definition keys
    */
   processDefinitionKeyIn: string;
+  /**
+   * A value which allows to filter tasks based on process definition Version Tag
+   */
+  processDefinitionVersionTag: string;
+  /**
+   * A JSON object used for filtering tasks based on process instance variable values. A property name of the object represents a process variable name, while the property value represents the process variable value to filter tasks by.
+   */
+  processVariables: object;
```

Comment:
```suggestion
  processVariables: Partial<T>;
```

also variables field should be: 

```
variables: (keyof T)[];
```

WDYT ?

Reply:
Yes, this is perfect.